### PR TITLE
Sync with twitch and improve releases

### PR DESCRIPTION
**Pull request by Emilgardis**

### CODE REVIEW

**Comment by Emilgardis** [comment]
I also want to implement https://github.com/twitch-rs/twitch_oauth2/pull/97 here

**Comment by Emilgardis** [comment]
need to lookover endpoints and see if I missed something else

**Comment by Nerixyz** [comment]
> need to lookover endpoints and see if I missed something else

`GetEventSubSubscriptionsRequest` now has an optional `user_id`: https://dev.twitch.tv/docs/api/reference#get-eventsub-subscriptions.

**Comment by Emilgardis** [comment]
fixed in [`ea7c91e` (#272)](https://github.com/twitch-rs/twitch_api/pull/272/commits/ea7c91ee245e132b8584a2a157e4167b55e76525#diff-ee10f5440b7a62234e05f4fe671435cd76a4d0745169b3db40122dda2ee06216)

**Comment by Emilgardis** [comment]
charity endpoint is in beta, should probably wait with 0ef8b97545da8bf49204dc6cc3963b0b7931e80c

**Comment by Emilgardis** [comment]
I''ll split out the charity and ci changes

File: src/helix/endpoints/raids/cancel_a_raid.rs [hunk 1]
```diff
@@ -0,0 +1,121 @@
+//!  Cancel a pending raid.
```
Comment (Emilgardis) [inline src/helix/endpoints/raids/cancel_a_raid.rs:1]:
```suggestion
//! Cancel a pending raid.
```

**Pull request merged**